README.md

```diff
@@ -41,8 +41,7 @@ This package relies on ONNX, NumPy, and ProtoBuf. If you are converting a model
 7. libsvm
 8. H2O
 
-ONNXMLTools has been tested with Python **2.7**, **3.5**, **3.6**, and **3.7**.  
-  `Note: some wrapped converters may not support python 2.x anymore.`
+ONNXMLTools has been tested with Python **3.5**, **3.6**, and **3.7**.  
```

Comment:
Can we add more details like "since onnxmltools 1.7, it stops the support python 2.x. "